lib/momento/cache_client.rb

```diff
@@ -446,6 +447,19 @@ def to_bytes(string)
       end
     end
 
+    def grpc_metadata(cache_name)
+      if @is_first_request
+        @is_first_request = false
+        {
+          cache: validate_cache_name(cache_name),
+          agent: "ruby:cache:#{VERSION}",
```

Comment:
Does it matter if agent and runtime-version are capitalized? They are in the other SDKs.

Reply:
yeah good call i might as well. I think it's case insensitive but we should try to be consistent.